coderd/provisionerdserver/provisionerdserver.go

```diff
@@ -271,10 +271,12 @@ func (s *server) AcquireJobWithCancel(stream proto.DRPCProvisionerDaemon_Acquire
 					Time:  dbtime.Now(),
 					Valid: true,
 				},
+				UpdatedAt: dbtime.Now(),
```

Comment:
nit: 2 calls to `Now()` might be separated by a few microseconds (or longer, if the goroutine is suspended in the middle), which might create an odd outcome where the job was updated later than it was completed